scripts/build.py

```diff
@@ -47,6 +47,7 @@
     'css', 'third_party.min.css')
 
 FONTS_RELATIVE_DIRECTORY_PATH = os.path.join('fonts', '')
+WEBFONTS_RELATIVE_DIRECTORY_PATH = os.path.join('webfonts', '')
```

Comment:
I think you need to add more comments throughout this file to explain the difference between "fonts" and "webfonts" dirs.

Reply:
Hi @seanlip  Added comments PTAL Thanks!